src/pol-vaults/mainnet.json

```diff
@@ -730,5 +730,10 @@
       "depositTokenAddress": "0xd7cfdd9514b6ef10e9cf271cf62955530adbc4ec",
       "slug": "yeet"
     }
+    {
+      "beraRewardVault": "0x11e13b70247e2bcc87544b366c47013ac51955a52",
```

Comment:
`0x11e13b70247e2bcc87544b366c47013ac51955a52` is not a valid address https://berascan.com/search?f=0&q=0x11e13b70247e2bcc87544b366c47013ac51955a52